checkov/common/runners/object_runner.py

```diff
@@ -203,7 +203,7 @@ def add_python_check_results(
                         bc_check_id=check.bc_id,
                         check_name=check.name,
                         check_result=result,
-                        code_block=self.definitions_raw[file_path][start - 1:end + 1],
+                        code_block=self.definitions_raw[file_path][start - 1:end + 2],
```

Comment:
why `end+2`?
won't this affect other code blocks too?

Reply:
It would affect all the yaml and json code blocks (that inherit this object runner).
I did this cause I noticed that when we do +1 with the way the lines are parsed we don't include in the results the closing bracket (which is usually a line below), so this would include always the next line.
But maybe in some yaml configs the next line is not related so I agree it might not be wanted, i'll revert it.
We could change the json parsing but im not sure its worth it, the current parsing is enough just looks a bit weird
great comment